affinity/cpp-20/dXXX1r0.md

```diff
@@ -195,58 +266,147 @@ Below *(Listing 2)* is an example of executing a parallel task over 8 threads us
 ```
 *Listing 2: Example of using the bulk_execution_affinity property*
```

Comment:
Your changes added a Listing 2 already; should we renumber or remove this one?

Reply:
I forgot to update the listing number, fixed.